dev/io.openliberty.jakarta.jaxb.3.0/src/jakarta/xml/bind/ModuleUtil.java

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
I assume this is a temporary overlay?

Reply:
@tevans78 that's correct, without an update to the ContextFactory to point to the Glassfish RI, the JPA feature will hit a NCDFE as the Jakarta JAX-B spec will keep trying to find the original Sun RI. I'll passing this change on to the Jakarta JAX-B project.